src/MCRun.jl

```diff
@@ -88,15 +112,21 @@ function swap_config_v!(mc_state,trial_config,dist2_mat_new,en_vec_new,en_tot)
     mc_state.en_tot = en_tot
     mc_state.count_vol[1] += 1
     mc_state.count_vol[2] += 1
+    
 end
 """
     acc_test!(ensemble, mc_state, new_energy, i_atom, trial_pos, dist2_new::Vector)  
 
-        (ensemble, nnp_state::NNPState, new_energy, i_atom, trial_pos)
+
+    acc_test!(ensemble, nnp_state::NNPState, new_energy, i_atom, trial_pos)
+
+    acc_test!(ensemble, mc_state, energy, i_atom, trial_pos, dist2_new::Vector,new_component_vector)
 
         The acc_test function works in tandem with the swap_config function, only adding the metropolis condition. Separate functions was benchmarked as very marginally faster. 
 
         The second method involes the NNPState subtype of the MCState, it follows the same basic functions as the other version, but accounts for the different output of the getenergy function. 
+        
+        The third method is used for the EmbeddedAtomPotential, since we need to have the components of that potential stored separately.
 
 NB: this method becomes redundant if we change the output and format of the getenergy function for the dimer potential: waiting on an email. 
 
```

Comment:
CAn you remind me what is this about?

Reply:
There is a vector of parameters rho and phi representing the two-body and density contributions of each atom, we need to pass the vector of __new__ parameters to the acc_test function because we need to exchange them for the old ones if the move is accepted.

Reply:
All seems fine then? I guess you still have to commit the changes?